kotlinx-coroutines-core/common/test/flow/internal/FlowScopeTest.kt

```diff
@@ -68,10 +68,10 @@ class FlowScopeTest : TestBase() {
             flowScope {
                 flowScope {
                     launch {
-                       throw CancellationException(null)
+                       throw CancellationException("")
```

Comment:
This is an intentional change, now common CE has both constructors from `Throwable?` cause and from `String?`, so `null` is ambiguous now